packages/mds-agency/request-handlers.ts

```diff
@@ -138,8 +138,7 @@ export const getVehicleById = async (req: AgencyApiRequest, res: AgencyApiRespon
   const { provider_id } = res.locals.scopes.includes('vehicles:read') ? req.query : res.locals
 
   log.info(`/vehicles/${device_id}`, cached)
-  const store = cached ? cache : db
-  const payload = await readPayload(store, device_id)
+  const payload = await readPayload(cached, device_id)
```

Comment:
The `cached` query string parameter is not part of the agency spec and should be removed. There should be one standard way to get a vehicle by device id.